packages/builder/src/builtin-tasks/console.ts

```diff
@@ -15,36 +16,39 @@ export default function (): void {
     .setAction(
       async (
         { noCompile }: { noCompile: boolean }, // eslint-disable-line
-        { config, run, runtimeArgs }           // eslint-disable-line
-
+        runtimeEnv: AlgobRuntimeEnv            // eslint-disable-line
```

Comment:
`// eslint-disable-line` can be removed here

Reply:
Will remove